Cargo.toml

```diff
@@ -78,5 +78,5 @@ sys-info = "0.9"
 [target."cfg(any(target_os = \"ios\", target_os = \"macos\"))".dependencies]
 cocoa = "0.24"
 core-graphics = "0.22"
-objc = { version = "0.2", features = [ "exception" ] }
+objc = { version = "0.2", features = ["exception"] }
```

Comment:
you still need to remove the feature here right? since it's enabled with default features now